Backend/app/schemas/schema.py

```
@@ -5,7 +5,7 @@
 class UserCreate(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)